cypress/plugins/index.js

```diff
@@ -15,7 +15,7 @@
 /**
  * @type {Cypress.PluginConfig}
  */
-module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
-}
+// module.exports = (on, config) => {
```

Comment:
Thanks for the recommendation. Let me see if I can try to "re-submit" this commit so that it doesn't include the changes I made in my other pull request, #81.

Reply:
Woohoo! I removed the changes made in #81 and all tests have passed.